File: Python/4-Array-based-Programs/54-move-all-negative-elements-to-one-side.py
arr = [-2, 1, 10, -6, -8, 5, -11]
j=0

for i in range(len(arr)):
    if arr[i] < 0:
        arr[i], arr[j] = arr[j], arr[i]
        j+=1
    
print("\nFinal Result: ", arr)

File: Java/4-Array-based-Programs/54-move-all-negative-elements-to-one-side.java
class Main {
    public static void main(String[] args) {
        int[] arr = { 8, -10, -5, 14, 6, -2, 9 };
        int j = 0;

        for (int i = 0; i < arr.length; i++) {
            if (arr[i] < 0) {
                int temp = arr[i];
                arr[i] = arr[j];
                arr[j] = temp;

                j++;
            }
        }
        System.out.println("Final Result is: ");
        for (int num : arr) {
            System.out.print(num + " ");
        }
    }
}